src/modules/uORB/SubscriptionCallback.hpp

```diff
@@ -53,6 +53,7 @@ class SubscriptionCallback : public SubscriptionInterval, public ListNode<Subscr
 	 * Constructor
 	 *
 	 * @param meta The uORB metadata (usually from the ORB_ID() macro) for the topic.
+	 * @param interval_ms The requested maximum update interval in milliseconds.
```

Comment:
0 to wait indefinitely, right? I'd probably add that.

Reply:
Applies to SubscriptionBlocking (a type of SubscriptionCallback), but yes.

Reply:
Done.